system-parachains/bridge-hub-paseo/src/lib.rs

```diff
@@ -159,20 +172,20 @@ impl_opaque_keys! {
 
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-    spec_name: create_runtime_str!("bridge-hub-polkadot"),
-    impl_name: create_runtime_str!("bridge-hub-polkadot"),
-    authoring_version: 1,
-    spec_version: 1_001_006,
-    impl_version: 0,
-    apis: RUNTIME_API_VERSIONS,
-    transaction_version: 4,
-    state_version: 1,
+	spec_name: create_runtime_str!("bridge-hub-paseo"),
```

Comment:
Unfortunately bridge hub was launch with the spec name `bridge-hub-polkadot`.
To keep this change in we have to be careful of upgrading the network by approving the new code with `system.authorizeUpgradeWithoutChecks`.

Nothing to block the changes from merging, but something to have into consideration when it comes to enact this new code.